lib/import-export-visitor.js

```diff
@@ -415,17 +414,11 @@ function getBlockBodyInfo(visitor, path) {
     // If parent is a Program, we may need to add a "use strict"
     // directive to enable const/let in Node 4.
     needToAddUseStrictDirective = visitor.enforceStrictMode;
-
     break;
 
   case "BlockStatement":
     body = parent.body;
-
-    if (hasOwn.call(parent, "start")) {
```

Comment:
It handles braceless statements like `if (condition) import "module"` by wrapping them with braces, since the block will need to contain both the variable declaration and the `module.importSync` call.

If we wanted to get really clever, I suppose we could include the `module.importSync` in the variable declaration, as the right-hand initializer for some dummy variable, but that might be more trouble than it's worth.